pi-manage

```diff
@@ -586,6 +586,25 @@ def create(name, scope, action):
     return r
 
 
+@policy_manager.command
+def create_from_file(name, filename):
+    """
+    create a new policy from a JSON-file
+    """
+    import json
+
+    f = open(filename, 'r')
+    contents = f.read()
+    f.close()
+    params = json.loads(contents)
```

Comment:
Please use ``ast.literal_eval(contents)``.
We are doing this when loading resolvers, too. I'd like to keep this consistent.
The literal_eval is a bit more powerful. e.g. It can also handle unicode keys in dicts, where json.loads fails.